tsconfig.json

```diff
@@ -1,12 +1,11 @@
 {
   "compilerOptions": {
-    "target": "es2015",
+    "target": "ES5",
     "module": "commonjs",
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
     "strict": true,
-    "keyofStringsOnly": true,
```

Comment:
field was deprecated, my editor didn't like it: https://www.typescriptlang.org/tsconfig#keyofStringsOnly